packages/@sanity/field/src/diff/components/ChangeTitleSegment.tsx

```diff
@@ -40,7 +53,13 @@ export function ChangeTitleSegment({
 
   // Changed/unchanged
   const readableIndex = (toIndex || 0) + 1
-  return <span className={styles.indexGroup}>#{readableIndex}</span>
+  return (
+    <Box padding={1}>
+      <Text size={1} weight="semibold">
+        #{readableIndex} {/* VIC NOTE: DONE */}
```

Comment:
What does `/* VIC NOTE: DONE */` mean?

Reply:
😄 oops, that's me reminding myself I already finished that. Will remove 👍